support-frontend/test/selenium/subscriptions/pages/SubsLandingPage.scala

```diff
@@ -8,7 +8,7 @@ class SubsLandingPage(implicit val webDriver: WebDriver) extends Page with Brows
 
   val url = s"${Config.supportFrontendUrl}/uk/subscribe"
 
-  private val header = cssSelector("#qa-component-subscriptions-by-country-group")
+  private val header = cssSelector(".subscriptions-landing-page")
```

Comment:
the "qa" prefix is so we know it's there for testing, although it didn't work in this case, it's a good technique to maintain in general I think